README.md

```diff
@@ -32,6 +32,10 @@ When `--prev-version` is included, the same thing happens but it starts from the
 
 By default, the text for the changelog entry is taken from the pull request title. If the pull request body includes `changelog:` on a separate line, it will override the default text.
 
+If `release-filter` is included, which should be a **regex** string, it will be used to filter out PRs by their names. Default value complies with semantic versioning that can optionally start with a *v*, like 1.0.1 or v1.0.1
```

Comment:
You say "it will be used to filter out PRs by their names". 
But it's filtering releases by their release tags :)

This makes me think that we should rename `--release-filter` to `--release-tag-filter` to be more explicit what the filter is for. More verbose, but easier to understand.
wdyt?